### ISSUE TITLE
ERROR: Something went wrong: Conversion failed! (dplay.se)

### ISSUE THREAD

**Issue by b0bber** [spaam/svtplay-dl#839]
Hey!
I've got some issues with the latest version of svtplay-dl..
When it tries to merge the audio and videosfiles from dplay it get an error..
```
INFO: Merge audio and video into far-vi-folja-med.s01e01.filip.fredrik.styr.kosan.mot.norrland-674-dplay.mp4
ERROR: Something went wrong: Conversion failed!
```
and I'm running 1.9.7-49-g70fafc7 on a linuxmachine, and i don't have any issues with tv4play.se, just dplay.. 

PS. both ffmpeg and avconv works with "-i file.m4a -i file.mp4 -c copy file.ts" if I try manually..

**Comment by myowncloudservice** [comment]
work OK for me!

I think you have an old version of svtplay-dl, you have to use the latest version v1.9.9
```

<RequestsCookieJar[]>
INFO: Selected to download hls, bitrate: 863
INFO: Outfile: far-vi-folja-med.s01e01.filip.fredrik.styr.kosan.mot.norrland-674-dplay.m4a
[266/266]`[====================================================================================================] ETA: 0:00:00
`INFO: Outfile: far-vi-folja-med.s01e01.filip.fredrik.styr.kosan.mot.norrland-674-dplay.mp4
[266/266]`[====================================================================================================] ETA: 0:00:00`
INFO: Merge audio and video into far-vi-folja-med.s01e01.filip.fredrik.styr.kosan.mot.norrland-674-dplay.mp4
INFO: Merging done, removing old files.
```

**Comment by b0bber** [comment]
Oh, okey, then I'll pull the latest git and compile it and try 👍 Thanx

**Comment by spaam** [comment]
comrade @crapmaster fixed this in #827 and its in master. will be in a release tonight.

**Comment by spaam** [comment]
this only happen on python2
